changelogs/fragments/179-allow-template-in-lookup-env.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - plugin_utils/turbo/lookup - fix issue when the ansible environment is set to a variable template instead of a dict
```

Comment:
```suggestion
  - plugin_utils/turbo/lookup - fix issue when the ansible environment is set to a variable template instead of a dict (https://github.com/ansible-collections/cloud.common/issues/179).
```